### ISSUE TITLE
sometimes making a new line and pressing tab inserts <tab>

### ISSUE THREAD

**Issue by trafnar** [imba/vscode-imba#91]
I can't reproduce this always, but sometimes I press the tab key to indent my code and a `<tab>` tag is inserted into my markup.

<img width="242" alt="image" src="https://user-images.githubusercontent.com/6732/122295679-5aec5080-ceae-11eb-9a19-6b888c21c7fe.png">

**Comment by trafnar** [comment]
v 3.0.2

**Comment by somebee** [comment]
This is very weird. Do you have some sort of funky vim mode or something where the tab-key would ever insert the characters tab? If not I really cannot understand what could cause this.

**Comment by trafnar** [comment]
Yeah, crossed my mind that it might not be imba related, but ive never
had it happen until using the new imba plugin, and it happened maybe 3
times. I’ll keep an eye out for it happening again and maybe try
disabling other extensions. I use the vscode vim extension too, possibly
related. 

On June 16, 2021, Kazuho Okui ***@***.***> wrote:
> This is very weird. Do you have some sort of funky vim mode or
> something where the tab-key would ever insert the characters tab? If
> not I really cannot understand what could cause this.
>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/imba/vscode-imba/issues/91#issuecomment-863174206>,
or unsubscribe <https://github.com/notifications/unsubscribe-
auth/AAABUTHQMYQSP7F4SUVEULDTTHOYXANCNFSM462JLC2A>.
